src/pocketmine/command/defaults/XpCommand.php

```diff
@@ -0,0 +1,98 @@
+use pocketmine\command\CommandSender;
+use pocketmine\command\utils\InvalidCommandSyntaxException;
+use pocketmine\lang\TranslationContainer;
+use pocketmine\utils\TextFormat;
+use pocketmine\Player;
+
+class XpCommand extends VanillaCommand{
+	
+        public function __construct(string $name){
+	        parent::__construct(
+		        $name,
+		        "%pocketmine.command.xp.description",
+		        "%pocketmine.command.xp.usage"
+		);
+	$this->setPermission("pocketmine.command.xp");
+	}
+        
+        public function execute(CommandSender $sender, string $commandLabel, array $args){
+	        if(!$this->testPermission($sender)){
+		        return true;
+	        }               
+                
+                if(count($args) < 1){
+	        throw new InvalidCommandSyntaxException();
+	        }
+                
+                if(isset($args[1])){
+                        $player = $sender->getServer()->getPlayer($args[1]);
+                        if($player === null){
+	                        $sender->sendMessage(new TranslationContainer(TextFormat::RED . "%commands.generic.player.notFound"));
+	                        return true;
+                        }
+                }
+                
+                if(strtolower(substr($args[0], -1)) == "l"){
+                        $lxp = $this->getInteger($sender, substr_replace($args[0], "", -1));
+                        if(isset($args[1])){
+                                $player->addXpLevels($lxp);
+                                Command::broadcastCommandMessage($sender, new TranslationContainer("%commands.xp.success.other", [
+                                        $lxp,
+                                        $player->getName()
+                                ]));
+                        }else{
+                                if(!($sender instanceof Player)){
+                                        throw new InvalidCommandSyntaxException;
+                                }
+                                $sender->getServer()->getPlayer($sender->getName())->addXpLevels($lxp);
+                                Command::broadcastCommandMessage($sender, new TranslationContainer("%commands.xp.success.self",[$lxp]));
+                        }
+                        return true;
+                }
+                
+                if(is_numeric($args[0])){
+                    $xp = $this->getInteger($sender, $args[0]);
+                        if(isset($args[1])){
+                                $player->addXp($xp);
+                                Command::broadcastCommandMessage($sender, new TranslationContainer("%commands.xp.success.other", [
+                                        $xp,
+                                        $player->getName()
+                                ]));
+                        }else{
+                            if(!($sender instanceof Player)){
+                                throw new InvalidCommandSyntaxException;
```

Comment:
nit: missing `()`

Reply:
what does `nit` mean?

Reply:
## @dktapps Yeah and would you please specify the issue here

Reply:
are you capable of reading?

> missing `()`

Reply:
@lukeeey it means small issue in this context.